api-scripts/patron/placing-holds/README.md

```diff
@@ -0,0 +1,38 @@
+## API test:
+EDS programmatically calls mod-patron using the API - 
+`POST /patron/account/{id}/instance/{instanceId}/hold`
+
+n example of an instance-level request to mod-patron - feel free to swap out this instanceId with a different one that could stress the system more by having a long list of items.
+
+`{
+    "item": {
+	    "instanceId": "c2063f7c-245a-4c24-b643-67fea831ebcc"
+	},
+    "requestDate": "2020-06-01T18:51:24.000+0000",
+    "pickupLocationId": "7068e104-aa14-4f30-a8bf-71f71cc15e07"
+}`
+ 
+
+
+## Modules required to be enabled as a pre-requisite to run JMeter script:
+##### Backend:
+- mod-patron-4.2.0-SNAPSHOT
+- mod-circulation-18.0.9
+- mod-circulation-storage-11.0.0
+- mod-inventory-storage-19.1.2
+- mod-inventory-14.1.3
+- mod-authtoken-2.4.0
+- mod-permissions-5.9.0
+- okapi-2.38.0
+
+- FOLIO release: FameFlower
+
+#Sql queries from jmeter-supported-data folder
+- users.csv (the list of users is decreased to 1000 users)<br/> 
+`psql -h pre-items-status-update-3-24-20-from-live.c5cqgppfbl8c.us-east-1.rds.amazonaws.com -d folio -U folio -t -A -F"," -c "SELECT CONCAT(id,'.', jsonb->>'barcode') FROM fs09000000_mod_users.users WHERE jsonb->>'barcode' != '';" > 'users.csv'
+`
+- holdings_instances_barcodes.csv (the list of users is decreased to 100 users<br/>
+`psql -h pre-items-status-update-3-24-20-from-live.c5cqgppfbl8c.us-east-1.rds.amazonaws.com -d folio -U folio -t -A -F"," -c "select RH.jsonb->>'instanceId' as InstanceID, I.jsonb->>'holdingsRecordId' as HoldingsRecordID, I.jsonb->>'barcode' as Barcode from fs09000000_mod_inventory_storage.item as I inner join fs09000000_mod_inventory_storage.holdings_record as RH ON I.jsonb->>'holdingsRecordId' = RH.jsonb->>'id' group by I.jsonb->>'holdingsRecordId', RH.jsonb->>'instanceId', I.jsonb->>'barcode';" > 'holdings_instances_barcodes.csv'
+`
+- service_points.csv <br/>
+they are received on UI
```

Comment:
Additionally, please explain how the script works. There is an IF controller, a Clean and Delete Request thread groups. A sentence or two explaining their roles in the script would be informative.